### DISCUSSION TITLE
AutoForm disabled props and useField

### DISCUSSION THREAD

**Discussion by simplecommerce** [vazco/uniforms#1093] [Q&A] [answered]
Hi,

I have a Checkbox field that I built using `useField(props.name, props, { absoluteName: true });`.
When I set `disabled` on the `AutoForm` component, it passes the `disabled` state correctly to my Checkbox.

In my current scenario, I need to disable the Checkbox but not the whole form, so I am passing `disabled={disabled}` to my component and it seems to disable it properly when true.

The issue I am having is when the `disabled` prop I am passing is `undefined`, it doesn't seem to recognize the `AutoForm` `disabled` prop no more, it only sees `undefined`.

Is this normal behavior?
Sorry if I am not being clear.

**Comment by radekmie** [comment, marked as answer]
This got solved in #1094.
